core/storage/exploration/gae_models.py

```diff
@@ -105,6 +117,19 @@ def _trusted_commit(
         """Record the event to the commit log after the model commit.
 
         Note that this extends the superclass method.
+
+        Args:
+            committer_id: String. The user_id of the user who committed the
+                change.
+            commit_type: String. Unique identifier of commit type. Possible
+                values are in COMMIT_TYPE_CHOICES in the base model.
```

Comment:
Refer to the constant's location explicitly, e.g. core.storage.blah1.blah2.COMMIT_TYPE_CHOICES